tempesta_fw/cfg.c

```diff
@@ -623,6 +623,9 @@ parse_cfg_entry(TfwCfgParserState *ps)
 		PFSM_COND_MOVE(ps->t == TOKEN_LITERAL, PS_MAYBE_EQSIGN);
 		FSM_COND_JMP(ps->t == TOKEN_SEMICOLON, PS_SEMICOLON);
 		FSM_COND_JMP(ps->t == TOKEN_LBRACE, PS_LBRACE);
+
+		ps->err = -EINVAL;
+		FSM_JMP(PS_EXIT);
```

Comment:
It seems there are similar places w/o proper handling of unexpected input. I.e. PS_MAYBE_EQSIGN - do we have the same issue in something like

        sticky name =

or

        srv_group sched= {
            .....

?